File: src/April.js
function April(que,poi,va,out,line){
    var query=que
    var point=poi
    var vari=va
    var output=out
    var i=line


    if(query[0][2]==1){
        if(output!=""){
        console.log(output)
        output="";
        }
        console.log("このバージョンはbeta版です。\n\nジョークを教えてください!")
    }else if(query[0][2]==2){
        var today = new Date();
        vari[point]=[0,today.getFullYear()]
    }else if(query[0][2]==3){
        var today = new Date();
        vari[point]=[0,today.getMonth() + 1]
    }else if(query[0][2]==4){
        var today = new Date();
        vari[point]=[0,today.getDate()]
    }else if(query[0][2]==5){
        var today = new Date();
        vari[point]=[0,today.getDay()]
    }else if(query[0][2]==6){
        var today = new Date();
        vari[point]=[0,today.getHours()]
    }else if(query[0][2]==7){
        var today = new Date();
        vari[point]=[0,today.getMinutes()]
    }else if(query[0][2]==8){
        var today = new Date();
        vari[point]=[0,today.getSeconds()]
    }else if(query[0][2]==9){
        var today = new Date();
        vari[point]=[0,today.getMilliseconds()]
    }else if(query[0][2]==10){
        var today = new Date();
        vari[point]=[0,(new Date(today.getFullYear(),today.getMonth(),today.getDate())-new Date(2000, 0, 1))/86400000]
    }

    return [point,vari,output,i]
}
module.exports={
    April
}

File: src/python/april.py
from datetime import datetime
from .utils import get_days_from_base

def april(query, point, vari, output, i):
    sub_cmd = int(query[0][2])
    today = datetime.now()
    
    if sub_cmd == 1:
        if output != "":
            output = ""
    elif sub_cmd == 2:
        vari[point] = [0, float(today.year)]
    elif sub_cmd == 3:
        vari[point] = [0, float(today.month)]
    elif sub_cmd == 4:
        vari[point] = [0, float(today.day)]
    elif sub_cmd == 5:
        js_day = (today.weekday() + 1) % 7
        vari[point] = [0, float(js_day)]
    elif sub_cmd == 6:
        vari[point] = [0, float(today.hour)]
    elif sub_cmd == 7:
        vari[point] = [0, float(today.minute)]
    elif sub_cmd == 8:
        vari[point] = [0, float(today.second)]
    elif sub_cmd == 9:
        vari[point] = [0, float(today.microsecond // 1000)]
    elif sub_cmd == 10:
        days = get_days_from_base(today.year, today.month, today.day)
        vari[point] = [0, days]
        
    return [point, vari, output, i]
